tests/component_handlers/helm_wrapper/test_helm_diff.py

```diff
@@ -18,11 +18,34 @@ def helm_diff(self) -> HelmDiff:
         return HelmDiff(HelmDiffConfig())
 
     def test_calculate_changes_unchanged(self, helm_diff: HelmDiff):
-        templates = [HelmTemplate(Path("a.yaml"), KubernetesManifest())]
+        templates = [
+            HelmTemplate(
+                Path("a.yaml"),
+                KubernetesManifest(
+                    **{
+                        "apiVersion": "v1",
+                        "kind": "Deployment",
+                        "metadata": {},
+                    }
```

Comment:
@disrupted 
This is pretty okay here, tbh. We use this initializing type in other places in the tests (look streams_app or producer_app tests). Besides, if we want to initialize the metadata like `MetaObject(a=1)` pyright throws an error.